var/spack/repos/builtin/packages/neuron/package.py

```diff
@@ -26,8 +26,11 @@ class Neuron(Package):
     # Patch which reverts 81a7a39 for numerical compat
     patch('revert_Import3d_numerical_format.patch', when='@7.8.0:')
 
+    # Patch which applies load balancing fix
+    patch('apply_79a4d2af_load_balance_fix.patch', when='@7.8.0:')
+
     version('develop', branch='master')
-    version('7.8.0a',  commit='92a208b', preferred=True)
+    version('7.8.0b',  commit='92a208b', preferred=True)
```

Comment:
`preferred` seems to be redundant, since the newest version is always preferred.